Getting 504s / crazy slow downs when borrowing from open library

### Summary

- **What is wrong?**
<!-- When was it noted, what is the issue, what is the scope? -->

<!-- Complete these sections once the problem is resolved. Be brief. -->

- **What caused it?**
<!-- What did investigation reveal to be the cause? -->

- **What fixed it?**
<!-- What fixed the problem? -->

- **Followup actions:**
<!-- Note any resulting changes or decisions --> 

### Steps to close
<!-- For instructions, please see: https://github.com/internetarchive/openlibrary/wiki/Disaster-History-Log#how-to -->
1. [ ] Assignment: Is someone assinged to this issue? (notetaker, responder)
2. [ ] Labels: Is there an `Affects:` label applied?
3. [ ] Diagnosis: Add a description and scope of the issue
4. [ ] Updates: As events unfold, is noteable **provenance** documented in issue comments? (i.e. useful debug commands / steps / learnings / reference links)
5. [ ] "What caused it?" - please answer in summary
6. [ ] "What fixed it?" - please answer in summary
7. [ ] "Followup actions:" actions added to summary

Sample error:

```
2020-04-24 21:48:48 [25346] [openlibrary.core.lending] [ERROR] failed to update ebook for matildadahl00dahl
Traceback (most recent call last):
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/openlibrary/core/lending.py", line 480, in sync_loan
    ebook.update(**kwargs)
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/openlibrary/core/lending.py", line 510, in update
    web.ctx.site.store[self['_key']] = self
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/infogami/infobase/client.py", line 567, in __setitem__
    return self._request(key, method='PUT', data=simplejson.dumps(data))
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/infogami/infobase/client.py", line 519, in _request
    out = self.conn.request(self.name, "/_store/" + path, method, data)
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/openlibrary/plugins/openlibrary/connection.py", line 53, in request
    return self.store_put(sitename, path, data)
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/openlibrary/plugins/openlibrary/connection.py", line 95, in store_put
    return self.conn.request(sitename, path, 'PUT', data)
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/openlibrary/plugins/openlibrary/connection.py", line 53, in request
    return self.store_put(sitename, path, data)
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/openlibrary/plugins/openlibrary/connection.py", line 95, in store_put
    return self.conn.request(sitename, path, 'PUT', data)
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/openlibrary/plugins/openlibrary/connection.py", line 53, in request
    return self.store_put(sitename, path, data)
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/openlibrary/plugins/openlibrary/connection.py", line 378, in store_put
    result = ConnectionMiddleware.store_put(self, sitename, path, data)
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/openlibrary/plugins/openlibrary/connection.py", line 95, in store_put
    return self.conn.request(sitename, path, 'PUT', data)
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/openlibrary/plugins/openlibrary/connection.py", line 537, in request
    return self.writer.request(sitename, path, method, data=data)
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/infogami/infobase/client.py", line 180, in request
    self.handle_error("%d %s" % (response.status, response.reason), response.read())
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/infogami/infobase/client.py", line 85, in handle_error
    raise ClientException(status, message, json)

```

More error

```
2020-04-24 22:15:24 [26210] [openlibrary.core.lending] [INFO] POST https://archive.org/services/openlibrary.php {'identifier': 'matildadahl00dahl', 'method': 'loan.sync'}
2020-04-24 22:15:54 [26210] [openlibrary.core.lending] [ERROR] POST failed
Traceback (most recent call last):
  File "/opt/openlibrary/deploys/openlibrary/openlibrary/openlibrary/core/lending.py", line 810, in _post
    timeout=config_http_request_timeout).read()
  File "/usr/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/usr/lib/python2.7/urllib2.py", line 404, in open
    response = self._open(req, data)
  File "/usr/lib/python2.7/urllib2.py", line 422, in _open
    '_open', req)
  File "/usr/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/usr/lib/python2.7/urllib2.py", line 1222, in https_open
    return self.do_open(httplib.HTTPSConnection, req)
  File "/usr/lib/python2.7/urllib2.py", line 1187, in do_open
    r = h.getresponse(buffering=True)
  File "/usr/lib/python2.7/httplib.py", line 1051, in getresponse
    response.begin()
  File "/usr/lib/python2.7/httplib.py", line 415, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python2.7/httplib.py", line 371, in _read_status
    line = self.fp.readline(_MAXLINE + 1)
  File "/usr/lib/python2.7/socket.py", line 476, in readline
    data = self._sock.recv(self._rbufsize)
  File "/usr/lib/python2.7/ssl.py", line 341, in recv
    return self.read(buflen)
  File "/usr/lib/python2.7/ssl.py", line 260, in read
    return self._sslobj.read(len)
SSLError: The read operation timed out
```

--- a/openlibrary/core/waitinglist.py
+++ b/openlibrary/core/waitinglist.py
@@ -263,6 +263,11 @@ def update_waitinglist(identifier):
     * When a book is checked out or returned
     * When a person joins or leaves the waiting list
     """
+    return None
+    # For books with many active loans, calls to loan.sync can be very slow / 504.
+    # It looks like, these two functions are handled on the IA side, and don't actually
+    # need to be called from Open Library. Disabling as a patch deploy for now; can
+    # likely remove remove in the near future.
     _wl_api.request("loan.sync", identifier=identifier)
     return on_waitinglist_update(identifier)
 

